Accelerated_Python_User_Guide/notebooks/Chapter_11_Distributed_Computing_cuPyNumeric.ipynb

```diff
@@ -53,7 +53,24 @@
    "source": [
     "## Basic installation\n",
     "\n",
-    "### Local linux server\n",
+    "### Local Linux server (pip)\n",
+    "\n",
+    "Python wheel packages for Python 3.10, 3.11, and 3.12 are available for AMD64 and ARM64 from [PyPI](https://pypi.org/project/nvidia-cupynumeric). To install these into an existing environment, run the command\n",
```

Comment:
Make it a little less susceptible to getting out of date
```suggestion
    "Python wheel packages are available for AMD64 and ARM64 from [PyPI](https://pypi.org/project/nvidia-cupynumeric). To install these into an existing environment, run the command\n",
```

Reply:
You will see installation errors if you don't use a supported Python version.

Reply:
https://github.com/NVIDIA/accelerated-computing-hub/pull/62/commits/b7e524f54338b4ee9a6c582bce2c30326f76fe56